src/main/java/com/rikaisan/mixin/BlockModelWireRedstoneMixin.java

```diff
@@ -22,67 +23,10 @@ public BlockModelWireRedstoneMixin(Block<T> block) {
 		super(block);
 	}
 
-	@Redirect(
-		method = "render(Lnet/minecraft/client/render/tessellator/Tessellator;III)Z",
-		at = @At(
-			value = "INVOKE",
-			target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z",
-			ordinal = 1
-		))
-	public boolean shouldConnectToWest(WorldSource worldSource, int x, int y, int z, int data) {
-		return AdditionalRedstoneWireLogic.shouldConnectToDiagonal(worldSource, x, y, z, data, Side.TOP);
-	}
-	@Redirect(
-		method = "render(Lnet/minecraft/client/render/tessellator/Tessellator;III)Z",
-		at = @At(
-			value = "INVOKE",
-			target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z",
-			ordinal = 3
-		))
-	public boolean shouldConnectToEast(WorldSource worldSource, int x, int y, int z, int data) {
-		return AdditionalRedstoneWireLogic.shouldConnectToDiagonal(worldSource, x, y, z, data, Side.TOP);
-	}
-	@Redirect(
-		method = "render(Lnet/minecraft/client/render/tessellator/Tessellator;III)Z",
-		at = @At(
-			value = "INVOKE",
-			target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z",
-			ordinal = 5
-		))
-	public boolean shouldConnectToNorth(WorldSource worldSource, int x, int y, int z, int data) {
-		return AdditionalRedstoneWireLogic.shouldConnectToDiagonal(worldSource, x, y, z, data, Side.TOP);
-	}
-	@Redirect(
-		method = "render(Lnet/minecraft/client/render/tessellator/Tessellator;III)Z",
-		at = @At(
-			value = "INVOKE",
-			target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z",
-			ordinal = 7
-		))
-	public boolean shouldConnectToSouth(WorldSource worldSource, int x, int y, int z, int data) {
-		return AdditionalRedstoneWireLogic.shouldConnectToDiagonal(worldSource, x, y, z, data, Side.TOP);
-	}
-
-	@Redirect(
-		method = "render(Lnet/minecraft/client/render/tessellator/Tessellator;III)Z",
-		slice = @Slice(
-			from = @At(
-				value = "INVOKE",
-				target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z",
-				ordinal = 8
-			),
-			to = @At(
-				value = "INVOKE",
-				target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z",
-				ordinal = 11
-			)
-		),
-		at = @At(
-			value = "INVOKE",
-			target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z"
-		)
-	)
-	public boolean shouldConnectToUp(WorldSource worldSource, int x, int y, int z, int data) {
-		return AdditionalRedstoneWireLogic.shouldConnectToDiagonal(worldSource, x, y, z, data, Side.BOTTOM);
+	@WrapOperation(method = "render", at = @At(value = "INVOKE", target = "Lnet/minecraft/core/block/BlockLogicWireRedstone;shouldConnectTo(Lnet/minecraft/core/world/WorldSource;IIII)Z"))
```

Comment:
Add explanation comment, the same as in `BlockLogicWireRedstoneMixin::fixDiagonal`.

Reply:
Fixed by 9f57f3c